### DISCUSSION TITLE
Disable client verification after update on third-party servers

### DISCUSSION THREAD

**Discussion by cabbagefork** [ppy/osu#20216] [Ideas] [answered]
I do not know if this is bug or a feature but this started happening in one of the latest updates.

Every single time an update gets pushed to my osu! client, I have to re-verify my client by logging in to my Bancho account after updating it. 

This normally is not an issue but I only play on third-party servers, such as Akatsuki!rx due to their relax score submission.

I'm forced to log out of my Akatsuki account, switch back back to Bancho, and then log back in to verify my client.

**Comment by peppy** [comment, marked as answer]
This is not related to lazer and also out of scope of stable.

If you are not using official servers, you need to report the issue to them.
